.docstr.yaml

```diff
@@ -0,0 +1,27 @@
+paths:
+  - tardis
+#badge: docs
+exclude: .*/test # regex
+verbose: 2 # int (0-3)
+skip_magic: True
+skip_file_doc: True
+skip_init: False
+skip_class_def: True
```

Comment:
I think this should be false. But could you post a list of these options (or a link to the description) in the ci-cd slack channel?

Reply:
https://github.com/HunterMcGushion/docstr_coverage/#options